scipy/signal/tests/test_filter_design.py

```diff
@@ -3611,6 +3611,83 @@ def test_fs_param(self):
         assert_allclose(abs(hp[2]), 1, rtol=1e-10)
 
 
+class TestIIRComb(object):
+    # Test erroneus input cases
+    def test_invalid_input(self):
+        # w0 is <= 0 or >= fs / 2
+        fs = 1000
+        with pytest.raises(ValueError, match='w0 must be between '):
+            iircomb(-fs, 30, 4, fs=fs)
+            iircomb(0, 35, 5, fs=fs)
```

Comment:
These need to each be in their own context manager for them all to execute (the first failure will kick you out of the context without executing the subsequent lines). You could for example loop over
```
for args in [(-fs, 30, 4), ...]:
    with pytest.raises(...):
        iircomp(*args, fs=fs)
```

Reply:
Fixed.